djangobb_forum/models.py

```diff
@@ -793,6 +810,16 @@ def filter_ham(self):
         pass
 
     @transition(
+        field=state, source=UNREVIEWED, target=USER_DELETED,
```

Comment:
I think we want to transition from all non-spam states, i.e., ``UNREVIEWED``, ``FILTERED_HAM``, ``MARKED_HAM``.